src/coreclr/CMakePresets.json

```diff
@@ -30,6 +30,21 @@
         }
       }
     },
+    {
+      "name": "osx-base",
+      "hidden": true,
+      "inherits": "base",
+      "cacheVariables": {
+        "CLR_CMAKE_TARGET_OS": "darwin"
+      },
+      "vendor": {
+        "microsoft.com/VisualStudioSettings/CMake/1.0": {
+          "hostOS": [
+            "macOS"
+          ]
```

Comment:
Visual Studio for Mac has retired and Visual Studio on Windows doesn't support mac. This should be some CLion specific setting, if any.

Reply:
Well, I realize that VS seems to hide non-Windows config based on `hostOS` value.

Reply:
Yeah, but I can update to CLion.